src/Model/BroadcastNotification.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Symbiote\Notifications\Model;
+
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Security\Group;
+use SilverStripe\Security\Member;
+use SilverStripe\Core\Injector\Injector;
+use Symbiote\Notifications\Service\NotificationService;
+use SilverStripe\Forms\ListboxField;
+
+
+
+class BroadcastNotification extends DataObject implements NotifiedOn
+{
+    private static $table_name = 'BroadcastNotification';
+
+    private static $db = [
+        'Title' => 'Varchar(255)',
+        'Content' => 'Text',
+        'SendNow' => 'Boolean',
+    ];
+
+    private static $many_many = [
+        'Groups' => Group::class
+    ];
+
+    public function onBeforeWrite()
+    {
+        if ($this->SendNow) {
+            $this->SendNow = false;
+            Injector::inst()->get(NotificationService::class)->notify(
+                'BROADCAST',
+                $this
+            );
+        }
+        parent::onBeforeWrite();
+    }
+
+    public function getCMSFields()
+    {
+        $fields = parent::getCMSFields();
+        
+        if ($this->ID) {
+            $fields->dataFieldByName('SendNow')->setRightTitle('If selected, this notification will be broadcast to all users');
```

Comment:
Should this be something like "If selected, this notification will be broadcast to all users in the following groups?" Or have a missed the override that sends to all?

Reply:
Yup, have updated that text